tests/.infra/crw-ci/pr-check/k8s/Jenkinsfile

```diff
@@ -218,14 +223,29 @@ pipeline {
             steps {
                 script {
                     echo "Create custom-resource patch file"
-                    sh """cat > $CUSTOM_RESOURCE_PATCH_FILE <<EOL
+                    if (cheSingleHost.toBoolean() == true) {
+                        sh """cat > $CUSTOM_RESOURCE_PATCH_FILE <<EOL
+spec:
+  server:
+    cheImage: '${cheImageRepo}'
+    cheImageTag: '${mutableCheImageTag}'
+    serverExposureStrategy: 'single-host'
+  k8s:  
+    singleHostExposureType: 'gateway'
+  auth:
+    updateAdminPassword: false
+EOL"""
+
+                    } else {
+                        sh """cat > $CUSTOM_RESOURCE_PATCH_FILE <<EOL
 spec:
   server:
     cheImage: '${cheImageRepo}'
     cheImageTag: '${mutableCheImageTag}'
```

Comment:
Another possible improvement: allow run of several workspaces at a time, which should make devfile tests more stable:
```
  server:
    customCheProperties:
      CHE_LIMITS_USER_WORKSPACES_RUN_COUNT: '-1'
```

Reply:
Done.